lib/vagrant-google/action/setup_winrm_password.rb

```diff
@@ -0,0 +1,81 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Changes:
+# April 2019: Modified example found here:
+# https://github.com/GoogleCloudPlatform/compute-image-windows/blob/master/examples/windows_auth_python_sample.py
+# to enable WinRM with vagrant.
+
+require "fog/google"
```

Comment:
Those requires should not be needed, can you take a look?

```
require "fog/google" 
require "googleauth"
require "google/apis/compute_v1"
require "vagrant/../../plugins/communicators/winrm/communicator"
```

Reply:
I missed the first 3 when I refactored to use fog-google directly.  For the others, things seem to work if I remove all of the requires.  I would have thought I'd need at least the following 2 because I make use of those Classes directly.

```
require "log4r"
require "vagrant/../../plugins/communicators/winrm/communicator"
```

At this point I've removed them all.